src/SignalR/common/Http.Connections/src/Internal/HttpConnectionContext.cs

```diff
@@ -95,7 +98,7 @@ public HttpConnectionContext(string id, IDuplexPipe transport, IDuplexPipe appli
 
         public DateTime LastSeenUtc { get; set; }
 
-        public HttpConnectionStatus Status { get; set; } = HttpConnectionStatus.Inactive;
```

Comment:
Where else is this setter called? It seems weird to protect state transitions in one place, but not elsewhere.

Reply:
Can we remove the setter now?

Reply:
No, it's used in `HttpConnectionContext.DisposeAsync` and it was a public API already, so we can't remove it.